### PR TITLE
Fixing bugs, adding title which is missing because of lack of nav

### PR DESCRIPTION
**Pull request by jakubvul-okta**
## Description:
- **What's changed?** 
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app-remediation/handle-remediation/index.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ title: Handle remediation
 
 Next you will need to handle the back-and-forth remediation process between your client and the Okta Identity Engine.
 
-The end result of this will be an interaction code, that in the <NextSectionLink>next step</NextSectionLink> will be exchanged for access and ID tokens.
+The end result of this will be an interaction code, that in the <GuideLink link="../get-tokens">next step</GuideLink> will be exchanged for access and ID tokens.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app-remediation/handle-remediation/index.md:7]:
```suggestion
The end result of this process is an interaction code, which in the <GuideLink link="../get-tokens">next step</GuideLink> is exchanged for access and ID tokens.
```

**Review by brianduffield-okta — APPROVED** [review]
Other than one suggestion, all looks good!

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app-remediation/before-you-begin/index.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-title: Before you begin
+title: Sign users in without redirection
 ---
 
 This guide shows you how to use Okta as the user store for your web application and sign users in using the interact code grant type.
 
 > **Note:** This functionality requires the usage of the Okta Identity Engine. This functionality is in general availability but is being gradually rolled out to customers. If you want to request to gain access to the Okta Identity Engine, please reach out to your account manager. If you do not have an account manager, please reach out to <oie@okta.com> for more information.
 
-#### Interact Code Overview
+#### Interaction Code Overview
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app-remediation/before-you-begin/index.md:9]:
```suggestion
#### Interaction Code overview
```

**Review by susanharper-okta — APPROVED** [review]
1 minor comment

**Pull request merged**